README.mkd

```diff
@@ -98,6 +98,14 @@ When localizing the strings found in R10k, follow the prescribed
 The workflow describes the rake tasks provided to generate the .po files for
 each locale.
 
+Releasing
+---------
+To release a new version of the r10k gem, ensure the [changelog](CHANGELOG.mkd) is up to date and open a pull request updating the [version file](lib/r10k/version.rb). When the PR is merged, a new release of the gem will be triggered.
+
+By default, a patch (Z) release will be triggered. To release a new major (X) or minor (Y) version, include `#major` or `#minor` (respectively) in your commit message to trigger the appropriate release.
```

Comment:
Does this mean that the version number from the PR is not actually tied to what gets released? I forget how our job works.

Reply:
Correct. Do you think I should call that out specifically? I went back and forth on it, worried it would be more info than people wanted/needed.

Reply:
Nah, don't need it, just wanted to remind myself.